crates/rust-analyzer/src/reload.rs

```diff
@@ -303,6 +303,12 @@ impl GlobalState {
         let files_config = self.config.files();
         let project_folders = ProjectFolders::new(&self.workspaces, &files_config.exclude);
 
+        #[cfg(not(windows))]
+        let standalone_server_name = "rust-analyzer-proc-macro-srv";
+
+        #[cfg(windows)]
+        let standalone_server_name = "rust-analyzer-proc-macro-srv.exe";
```

Comment:
Have you seen https://doc.rust-lang.org/std/env/consts/constant.EXE_SUFFIX.html?

Reply:
Looks neat! I'm stealing that.